src/routes/templates/templates.json

```
@@ -1,4 +1,14 @@
 [
+	{
+		"title": "SvelteKit Authentication Example Project",
+		"url": "https://github.com/nstuyvesant/sveltekit-auth-example",
+		"description": "An example of how to manage local and Sign in with Google Authentication using SvelteKit and a PostgreSQL database. This project has maintained compatibility through the breaking changes in SvelteKit.",
+		"npm": "sveltekit-auth-example",
+		"addedOn": "2022-10-02",
+		"category": "SvelteKit",
+		"tags": ["templates", "typescript", "authentication", "postgresql", "google"],
```

```suggestion
		"tags": ["templates", "typescript", "authentication", "database"],
```

- `postgresql` is not an existing tag, but `database` already exist
- `google` is not an existing tag
- `authentication` is not an existing tag, but seem legit to me to be added